data/remnant/remnant 2 side missions.txt

```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+				decline
+			label celebrationdefinition
+			`	"The Celebration of Remembrance is a gathering; we take a moment to remember where we have come from, appreciate where we are, and express hope for where we will be in the future." Her gestures take on a critical tone. "That sounds very formal, but it is more artistic than defined." She glances at the slowing flow of people. "It will be starting shortly, and I want to attend this month, so I should get moving. Would you like to join us?"`
+			choice
+				`	"Yes, I do."`
+				`	"No thanks."`
+					goto celebrationdeclined
+			label celebrationattendance
+			`	As the two of you join the flow of people, Taely gestures once again. "If you would like, I could provide you with some commentary and explanation, but it is more about feeling than doing anything specific. Together, we remember and celebrate through free-form dance mixed with singing and music. You are welcome to participate, but try not to disrupt or impede others in the way you join the dancing."`
+			choice
+				`	"No, that is alright. I will just experience it."`
+				`	"Yes, that would be appreciated."`
+					goto celebrationannotated
+			`	The flow of people you are a part of joins with other streams, finally leading out of the typical hallways and into a large cave. A large opening on one side opens upwards towards the night sky, framed by rough stone and a handful of shield emitters. Stalagmites ring the perimeter of the room with matching stalactites above them, but there is no trace of drips or moisture here. While the walls are rough, natural stone, the floor and stalagmites have been polished smooth. At a quick glance, you estimate that there are probably over a hundred people in here with more filtering in through several doors. Taely leads you through the crowd, frequently gesturing greetings to those she passes by.`
+			`	Without warning the room stills, and the doors close. You see a flurry of gestures wash over the room, as if everyone is repeating what someone closer to the center is saying, but you cannot see anyone starting it. "Perimeter secure?" it asks, followed by a ripple of gestures from the edges of the room back toward the center: "All clear."`
+			`	Suddenly you are aware of a drum beat building in the silence, and realize that it had started with the first ripple of gestures, so quiet that it was unnoticeable. The beat builds gradually, a slow, steady rhythm.`
+			`	"Thump."`
+			`	"Thump."`
+			`	"Thump."`
+			`	"Thump."`
+			choice
+				`	(Look around the room.)`
+				`	(Move in time with the beat.)`
+			`	The beat echoes in the cavern, but you quickly manage to locate the source: an elderly Remnant using her hands to thump on a polished stalagmite. Judging by the resonance, it appears that it must be hollow. As you watch, a child steps up next to her and starts hitting the rock with their palms, slightly off-beat, changing the pattern into a hauntingly familiar one:`
```

Comment:
Are hollow stalagmites a thing in real life?
It seems hollow stalactites are a thing, but not so much stalagmites?
Is the implication that the dripping that usually forms these happened upwards, for some reason, I guess someone was screwing with gravity?
Or, something else? Are they just a different type to what we might have on Earth? Are they supposed to be artificial?

Reply:
Go with the simplest explanation: They're artificially hollowed out. I'm not actually averse to the other options (such as someone messing with gravity or that they are somehow naturally like that), but simply that they've been hollowed out artificially somehow is the main thing.